<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ProjectCompileParamatersWidget</class>
 <widget class="QWidget" name="ProjectCompileParamatersWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>675</width>
    <height>447</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="grpAllowParallelBuilding">
     <property name="title">
      <string>Parallel Build</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Parallel Jobs(0 means infinite):</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="spinParallelJobs"/>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QTabWidget" name="tabCommands">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tabCCompiler">
      <attribute name="title">
       <string>C Compiler</string>
      </attribute>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <property name="leftMargin">
        <number>5</number>
       </property>
       <property name="topMargin">
        <number>5</number>
       </property>
       <property name="rightMargin">
        <number>5</number>
       </property>
       <property name="bottomMargin">
        <number>5</number>
       </property>
       <item>
        <widget class="QPlainTextEdit" name="txtCCompiler"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabCppCompiler">
      <attribute name="title">
       <string>C++ Compiler</string>
      </attribute>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <property name="leftMargin">
        <number>5</number>
       </property>
       <property name="topMargin">
        <number>5</number>
       </property>
       <property name="rightMargin">
        <number>5</number>
       </property>
       <property name="bottomMargin">
        <number>5</number>
       </property>
       <item>
        <widget class="QPlainTextEdit" name="txtCPPCompiler"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabLinker">
      <attribute name="title">
       <string>Linker</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <property name="leftMargin">
        <number>5</number>
       </property>
       <property name="topMargin">
        <number>5</number>
       </property>
       <property name="rightMargin">
        <number>5</number>
       </property>
       <property name="bottomMargin">
        <number>5</number>
       </property>
       <item>
        <widget class="QPlainTextEdit" name="txtLinker"/>
       </item>
       <item>
        <widget class="QFrame" name="frame">
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QToolButton" name="btnChooseLib">
            <property name="text">
             <string>Add Library Files</string>
            </property>
            <property name="toolButtonStyle">
             <enum>Qt::ToolButtonTextBesideIcon</enum>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabResource">
      <attribute name="title">
       <string>Resource</string>
      </attribute>
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <property name="leftMargin">
        <number>5</number>
       </property>
       <property name="topMargin">
        <number>5</number>
       </property>
       <property name="rightMargin">
        <number>5</number>
       </property>
       <property name="bottomMargin">
        <number>5</number>
       </property>
       <item>
        <widget class="QPlainTextEdit" name="txtResource"/>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>grpAllowParallelBuilding</tabstop>
  <tabstop>spinParallelJobs</tabstop>
  <tabstop>tabCommands</tabstop>
  <tabstop>txtCCompiler</tabstop>
  <tabstop>txtCPPCompiler</tabstop>
  <tabstop>txtLinker</tabstop>
  <tabstop>btnChooseLib</tabstop>
  <tabstop>txtResource</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
